src/app/core/services/field-config.service.ts

```diff
@@ -10,6 +10,28 @@ declare var $: any;
 
 
 export class FieldConfigService {
+  static KEY_CONDITIONS: String[] = [
```

Comment:
this array (and `VAL_CONDITION`) are used when we initialize forms to translate the numeric codes in the preset config into strings.